mobile/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -444,11 +444,9 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = Runner/Runner.entitlements;
 				CODE_SIGN_IDENTITY = "Apple Development";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_STYLE = Automatic;
```

Comment:
Please don't commit this.

Reply:
have you read the description of this PR? it's just a draft for aiding debugging of the problem we're having 🙃 none of it gets in.

Reply:
I have changed the name of the PR to make it less ambiguous, I realised that it would have helped to filter out the PR before you jumped into it.